frontend/src/metabase/visualizations/components/settings/ChartSettingFieldPicker.styled.tsx

```diff
@@ -46,8 +46,8 @@ export const ChartSettingFieldPickerRoot = styled.div<ChartSettingFieldPickerRoo
     color: ${color("text-dark")};
   }
 
-  ${SelectButton.Root} {
-    ${props => props.disabled && `background-color: ${color("white")};`}
+  ${SelectButton.Root}:disabled {
```

Comment:
👍  I think this is preferable. As `SelectButton.Root` is essentially a `button` not a custom React component.